synapse/lib/stormlib/stix.py

```diff
@@ -350,7 +359,7 @@ def uuid4(valu=None):
                     },
                     'rels': (
                         # TODO
-                        # ('has', 'vulnerability', ''),
+                        ('targets', 'vulnerability', '-> risk:hasvuln -> risk:vuln'),
```

Comment:
There is not a STIX relationship which actually records that a particular tool has a vulnerability.

Reply:
This pivot is also incorrect. This would document that each tool targets each vuln that it (itself) is vuln to.